festival/src/main/java/com/hubses/festival/web/rest/CustomerResource.java

```diff
@@ -18,34 +24,88 @@
     private UserService userService;
 
     @RequestMapping("")
-    public Iterable<User> getAllCustomers() {
-        return userService.getAllCustomers().map(customers -> customers)
+    public Iterable<CustomerModelDTO> getAllCustomers() {
+        return userService.getAllCustomers().map(customers -> {
+            List<CustomerModelDTO> userList = new ArrayList<>();
+            for (User customer : customers) {
+                userList.add(CustomerModelDTO.builder()
+                        .id(customer.getId())
+                        .username(customer.getUsername())
+                        .name(customer.getName())
+                        .surname(customer.getSurname())
+                        .role(customer.getRole().toString())
+                        .events(customer.getEvents().stream().map(Event::getName).collect(Collectors.toList()))
+                        .build());
+            }
+            return userList;
+        })
                 .orElse(new ArrayList<>());
     }
 
     @RequestMapping("/{id}")
-    public User getCustomer(@PathVariable(value = "id") long id) {
-        return userService.getCustomerById(id).map(customer -> customer)
-                .orElse(new User());
+    public CustomerModelDTO getCustomer(@PathVariable(value = "id") long id) {
+        return userService.getCustomerById(id).map(customer ->
+                CustomerModelDTO.builder()
+                        .id(customer.getId())
+                        .username(customer.getUsername())
+                        .name(customer.getName())
+                        .surname(customer.getSurname())
+                        .role(customer.getRole().toString())
+                        .events(customer.getEvents().stream().map(Event::getName).collect(Collectors.toList()))
+                        .build())
+                .orElse(new CustomerModelDTO());
     }
 
+    @Transactional
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
-    public User deleteCustomer(@PathVariable(value = "id") long id) {
-        return userService.deleteCustomerById(id).map(customer -> customer)
-                .orElse(new User());
+    public CustomerModelDTO deleteCustomer(@PathVariable(value = "id") long id) {
```

Comment:
According to REST you might return 204 if customer is deleted, why return empty customer?